base/applications/network/tracert/tracert.cpp

```diff
@@ -153,23 +159,29 @@ Usage()
     OutputText(IDS_USAGE);
 }
 
-static ULONG
+static bool
 GetULONG(
-    _In_z_ LPWSTR String
+    _In_ PWSTR String,
+    _Out_ ULONG *Value
 )
 {
-    ULONG Length;
-    Length = wcslen(String);
+    LPWSTR StopString;
+    
+    // check input arguments
+    if (String == NULL || Value == NULL || *String == UNICODE_NULL)
```

Comment:
```suggestion
    if (*String == UNICODE_NULL)
```
As with the other function, the parameters aren't optional, so the function won't be called with NULL pointers, unless the caller doesn't know what they are doing.